### PR TITLE
Remove: ExodusMovement.Exodus version 22.6.17

### PR DESCRIPTION
**Pull request by coolplaylinbot**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121767)

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
@stephengillie Please recheck these PRs carefully. The backend isn't always perfect.

File: manifests/e/ExodusMovement/Exodus/22.6.17/ExodusMovement.Exodus.installer.yaml
Comment (CoolPlayLin) [inline manifests/e/ExodusMovement/Exodus/22.6.17/ExodusMovement.Exodus.installer.yaml:1]: Mistake Pull Request

File: manifests/e/ExodusMovement/Exodus/22.6.17/ExodusMovement.Exodus.locale.en-US.yaml
Comment (CoolPlayLin) [inline manifests/e/ExodusMovement/Exodus/22.6.17/ExodusMovement.Exodus.locale.en-US.yaml:1]: Mistake Pull Request

File: manifests/e/ExodusMovement/Exodus/22.6.17/ExodusMovement.Exodus.yaml
Comment (CoolPlayLin) [inline manifests/e/ExodusMovement/Exodus/22.6.17/ExodusMovement.Exodus.yaml:1]: Mistake Pull Request

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
> Mistake Pull Request

**Pull request closed without merging**